src/components/Feedback/index.tsx

```diff
@@ -1,15 +1,15 @@
 import { useState } from "react";
 import styled from "@emotion/styled";
 import IconButton from "@leafygreen-ui/icon-button";
-import { uiColors } from "@leafygreen-ui/palette";
+import { palette } from "@leafygreen-ui/palette";
 import Cookies from "js-cookie";
 import Icon from "components/Icon";
 import { StyledLink as Link } from "components/styles";
 import { HIDE_FEEDBACK } from "constants/cookies";
 import { size } from "constants/tokens";
 import { useSpruceConfig } from "hooks";
 
-const { green } = uiColors;
+const { green } = palette;
```

Comment:
I think we should change the `green.base` => `green.dark1` in this file! the megaphone is hard to see